tmva/pymva/test/EmitFromPyTorch.cxx

```diff
@@ -1,46 +0,0 @@
-// Author: Sanjiban Sengupta, 2021
```

Comment:
We can maybe do this with Keras as well later?

Reply:
yes we could do this, but we need to extend RSofieReader to support multiple-inputs